link-verifier/fileTests/badFiles/fileWithMDIntheNameAndABrokenLink.md

```diff
@@ -0,0 +1,11 @@
+# I am a test file to make sure the regex name replacement only replaces files
+# that end in .md, not all files with md in their name.
```

Comment:
Why do we want name replacement for files with `md` in the name?

Reply:
We do not.
It was a bug in the link verifier that it was replacing this in filenames.
After fixing the bug I made this test to ensure that this issue didn't happen again.